subsys/mgmt/Kconfig

```diff
@@ -11,6 +11,7 @@ config MCUMGR_SMP_BT
 	select BT
 	select BT_PERIPHERAL
 	select BT_GATT_DYNAMIC_DB
+	select LEGACY_TIMEOUT_API
```

Comment:
This is already selected in the `BT` kconfig file here `zephyr/subsys/bluetooth/Kconfig`

Reply:
right, oversaw

Reply:
thanks to https://github.com/zephyrproject-rtos/zephyr/pull/24341 mcumgr library supports new timer API now.